config.yml

```diff
@@ -779,6 +945,265 @@ production_factories:
     repair_inputs:
       Diamond:
         material: DIAMOND
+  Random_Wooden_Tool_Factory:
+    name: Random Wooden Tool Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:  
+      Sticks:
+        material: STICK
+        amount: 96   
+    recipes:
+      - Create_Wooden_Axe
+      - Create_Wooden_Spade
+      - Create_Wooden_Axe
+      - Create_Fishing_Rod

```

Comment:
No Wooden hoes?

I'm guessing it's just a lack of enchantment so no need for them to be here, correct?